components/com_content/src/View/CategoryForm/HtmlView.php

```diff
@@ -0,0 +1,201 @@
+<?php
+
+/**
+ * @package     Joomla.Site
+ * @subpackage  com_content
+ *
+ * @copyright   (C) 2024 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Component\Content\Site\View\CategoryForm;
+
+use Joomla\CMS\Factory;
+use Joomla\CMS\Language\Multilanguage;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\MVC\View\GenericDataException;
+use Joomla\CMS\MVC\View\HtmlView as BaseHtmlView;
+use Joomla\CMS\Router\Route;
+use Joomla\Component\Content\Site\Helper\RouteHelper;
+use Joomla\Component\Content\Site\Model\CategoryFormModel;
+
+// phpcs:disable PSR1.Files.SideEffects
+\defined('_JEXEC') or die;
+// phpcs:enable PSR1.Files.SideEffects
+
+/**
+ * HTML Category Form View class for the Content component.
+ *
+ * @since  6.1.0
```

Comment:
```suggestion
     * @since   __DEPLOY_VERSION__
```

doing this means our build scripts will correctly set the version - it might not make it to 6.1 ;)